docs/git-integration.md

```diff
@@ -0,0 +1,34 @@
+# Git Integration for DevOps & GitOps
+
+Git is integrated into Monokle to tackle GitOps seamlessly within the application. If you work within a DevOps culture our Git integration provides advantages for you and your team by keeping a history of changes made to your applications stored in Git to ensure access to a reliable audit trail of your changes.
```

Comment:
Monokle integrates with Git to provide support for Git based flows, including GitOps. If you using DevOps techniques and culter, our Git integratrion provides a history of changes made to your applications, with a full audit trail of your changes.